databases/week1/homework.sql

```diff
@@ -0,0 +1,29 @@
+/* 1. Find out how many tasks are in the task table */
+SELECT COUNT(id) FROM task;
+
+/* 2. Find out how many tasks in the task table do not have a valid due date */
+SELECT COUNT(id) FROM task WHERE due_date IS NULL;
+
+/* 3. Find all the tasks that are marked as done */
+SELECT title FROM task LEFT JOIN status ON task.status_id = status.id WHERE status.name = 'Done';
+
+/* 4. Find all the tasks that are not marked as done */
+SELECT title FROM task LEFT JOIN status ON task.status_id = status.id WHERE status.name != 'Done';
```

Comment:
When you have more then one table in thee query, it is a good practice to use the table aliases in the select. 
- it does the query more readable 
- and it prevents error in the case when both tables have the same field
```suggestion
SELECT task.title FROM task LEFT JOIN status ON task.status_id = status.id WHERE status.name != 'Done';
```